prow/external-plugins/cherrypicker/server.go

```diff
@@ -403,25 +425,37 @@ func (s *Server) handle(l *logrus.Entry, requestor string, comment github.IssueC
 	if err != nil {
 		resp := fmt.Sprintf("new pull request could not be created: %v", err)
 		s.log.WithFields(l.Data).Info(resp)
-		return s.ghc.CreateComment(org, repo, num, plugins.FormatICResponse(comment, resp))
+		return s.createComment(org, repo, num, comment, resp)
 	}
 	resp := fmt.Sprintf("new pull request created: #%d", createdNum)
 	s.log.WithFields(l.Data).Info(resp)
-	if err := s.ghc.CreateComment(org, repo, num, plugins.FormatICResponse(comment, resp)); err != nil {
+	if err := s.createComment(org, repo, num, comment, resp); err != nil {
 		return err
 	}
 	if !s.prowAssignments {
-		if err := s.ghc.AssignIssue(org, repo, createdNum, []string{comment.User.Login}); err != nil {
+		login := requestor
+		if comment != nil {
+			login = comment.User.Login
+		}
```

Comment:
nit: I don't think this is necessary. The requestor will already be the comment author if the request came from a comment.

Reply:
Indeed. Fixed.